pages/collective-page.js

```diff
@@ -160,8 +171,8 @@ class CollectivePage extends React.Component {
     this.setState({ showOnboardingModal: bool });
   };
 
-  getCanonicalURL(slug) {
-    return `${process.env.WEBSITE_URL}/${slug}`;
+  getCanonicalURL(collective) {
+    return `${process.env.WEBSITE_URL}/${getCollectivePageRoute(collective)}`;
```

Comment:
Don't we end up with a double `/`? Like `opencollective.com//babel`?

Reply:
I've changed this part to a simple string concatenation so we don't have this issue. One thing I also changed was the return of `#` if an empty `collective` object is passed; 

https://github.com/opencollective/opencollective-frontend/blob/7899d4247116632d7b0e27ea06c02966a399f042/lib/url-helpers.js#L153-L154

I think we need to return empty string instead here as otherwise we are replying on the fact that the url string that precedes this part ends with `/`.